CharacterClient/src/config.py

```
@@ -0,0 +1,68 @@
+# --- Logs Path ---
+# Default is '[CLIENT_DATA_PATH]/logs/'.
+CLIENT_LOGS_PATH = os.path.join(CLIENT_DATA_PATH, "logs")
+
+# --- Temporary Audio Path ---
+# For storing synthesized audio before sending to server, or other temp files.
+# Default is '[CLIENT_DATA_PATH]/temp_audio/'.
+CLIENT_TEMP_AUDIO_PATH = os.path.join(CLIENT_DATA_PATH, "temp_audio")
+
+
+# --- Function to Create Directories ---
+def ensure_client_directories():
```

Ruff (0.11.9)

Missing return type annotation for public function `ensure_client_directories`

Add return type annotation: `None`

(ANN201)